src/v3/src/components/ImageLink/ImageLink.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box } from '@mui/material';
+import { Link } from '@okta/odyssey-react-mui';
+
+import { ImageLinkElement, UISchemaElementComponent } from '../../types';
+
+const ImageLink: UISchemaElementComponent<{
+  uischema: ImageLinkElement
+}> = ({ uischema }) => {
+  const { options } = uischema;
+  const Icon = options.svgIcon;
+  return (
+    <Box
+      display="flex"
+      justifyContent={options.alignment}
+      alignItems="center"
+      flexWrap="wrap"
+    >
+      <Link
+        href={options.url}
+        rel="noopener noreferrer"
+        data-se={options.dataSe}
```

Comment:
```suggestion
        testId={options.dataSe}
```
Odyssey maps the `testId` prop to `data-se`

Reply:
Apologies @vakaevich-okta , I realize I've been some suggested changes assuming that `@okta/odyssey-react-mui` was already at version 1.x+, but your branch is still on 0.x. These changes will work out once your branch is merged into master since we've now merged in the 1.x changes, but my suggestions have probably led to some confusion for you